workspace/dry_clone.py

```
@@ -0,0 +1,72 @@
+            env["GIT_USERNAME"] = "oauth2"
+
+        proc = await asyncio.wait_for(
+            asyncio.create_subprocess_exec(
+                *cmd,
+                stdout=asyncio.subprocess.PIPE,
+                stderr=asyncio.subprocess.PIPE,
+                env=env,
+            ),
+            timeout=timeout,
+        )
+        stdout, stderr = await proc.communicate()
```

Ruff (0.15.12)

[warning] Unpacked variable `stdout` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)